src/serac/numerics/functional/element_restriction.cpp

```diff
@@ -381,7 +381,7 @@ axom::Array<DoF, 2, axom::MemorySpace::Host> GetFaceDofs(const mfem::FiniteEleme
     return axom::Array<DoF, 2, axom::MemorySpace::Host>{};
   } else {
     uint64_t                                     dofs_per_face = face_dofs.size() / n;
-    axom::Array<DoF, 2, axom::MemorySpace::Host> output({n, dofs_per_face});
+    axom::Array<DoF, 2, axom::MemorySpace::Host> output(n, dofs_per_face);
```

Comment:
```suggestion
    axom::Array<DoF, 2, axom::MemorySpace::Host> output({n, dofs_per_face});
```

Reply:
out of curiosity, why does this change keep showing up in every pr and getting reverted?

Reply:
I updated axom in my local environment at some point (but haven't pushed that changed through a PR). The initializer list constructor for `axom::Array` no longer exists in develop.